.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
```

Comment:
`bandit` is indeed a good addition. Maybe it would make sense to run it twice: once for actual code and once for the tests directory?

Or, if it's possible to configure skipped checks individually for each module (or even class/function), I'd opt for that. Do you @cclauss know whether that's supported in `bandit`?

Also:
- `B101` shouldn't be skipped outside tests, since there shouldn't be any asserts in the actual package
- Darker relies on `subprocess` to run Git. `B404` and `B603` should be skipped in some modules.
- We do have this in `darker/tests/test_linting.py`:
  ```python
  # Use evil `eval()` so we get Windows compatible expected paths:
  # pylint: disable=eval-used
  assert result == [
      eval(f'f"{line}"', {"git_repo": git_repo}) for line in expect_output
  ]
  ```
  and it triggers `B307` which recommends `ast.literal_eval()`. This is a good heads-up and should probably be looked into. Would you @cclauss happen to have time to think about a better alternative for this test case while keeping Windows compatibility?

Reply:
I replaced the `eval` and addressed also all the other remarks from `bandit`.